### PR TITLE
DNM - Python3

### PR DESCRIPTION
**Pull request by goneri**
Depends-On: https://github.com/mattclay/aws-terminator/pull/239

### CODE REVIEW

File: aws/terminator.yml [hunk 1]
```diff
@@ -52,34 +52,34 @@
       pip:
         requirements: "{{ playbook_dir }}/requirements.txt"
         virtualenv: "{{ packaging_dir }}/terminator-requirements/python"
-        virtualenv_python: python3.7
+        virtualenv_python: python3.9
```
Comment (abikouo) [inline aws/terminator.yml:55]: we should rather define this in a dedicated to simplify code updates, I have done it in another PR

**Review by abikouo — APPROVED** [review]

File: aws/terminator.yml (hunk 1 above)
Comment (goneri) [reply aws/terminator.yml:55]: Oh sorry for the confusion, this PR is not ready for review. It's a series of commits that I want to submit in independent PR.
